doc/script_commands.txt

```diff
@@ -7039,6 +7039,18 @@ NPCs talking while hidden then revealing... you can wonder around =P).
 
 ---------------------------------------
 
+*cloakonnpc("<NPC object name>"{, <AID>})
+*cloakoffnpc("<NPC object name>"{, <AID>})
```

Comment:
```suggestion
*cloakoffnpc("<NPC object name>"{, <account_id>})
```
instead of `AID` , perhaps `account_id` a better choice, just like other? easier to newbie to understand what it is?